File: Code/BackEnd/Libs/Repositories/Models/TransferWorkflowStatusHistory.cs
using System;

namespace Repositories.Models;

public partial class TransferWorkflowStatusHistory : AuditableEntity
{
    public int Id { get; set; }

    public int TransferWorkflowId { get; set; }

    public int? FromStatus { get; set; }

    public int ToStatus { get; set; }

    public string? Note { get; set; }

    public virtual TransferWorkflow TransferWorkflow { get; set; } = null!;
}

File: Code/ReactUI/src/types/transferWorkflowStatusHistory.ts
export type TransferWorkflowStatusHistory = {
  id: number;
  transferWorkflowId: number;
  sourceAddressId: number;
  destinationAddressId: number;
  fromStatus: number | null;
  toStatus: number;
  note: string | null;
  createdAtUtc: string;
  createdBy: string;
};